docs/source/usage.rst

```diff
@@ -95,6 +98,22 @@ to conserve storage space: ::
 This exclude pattern will skip all restart subdirectories under the short-term archive,
 except for those with years ending in '0' or '5'.
 
+Example with Globus
```

Comment:
Would this be a good place to add a note about first time configuration (API key?). 

- Suggest that users first to a small transfer to test functionality.
- Remind users to first activate Globus endpoints via Globus web interface.
-

Reply:
It might be good to include a `note` box or 2, e.g.:

```
.. note::
    If you are a new Globus user, you should first do a small transfer to test functionality.

.. note::
    Always activate Globus endpoints via the Globus web interface before running ``zstash``.
```

Reply:
The suggested notes have been added https://github.com/E3SM-Project/zstash/pull/154/commits/7498fcf311a33d4a663e9c0790ebf54cbcdb2875.